client/suite_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 The Metal Authors.
```

Comment:
We dont use this license text anymore. Could you please adjust it similar to the other files ?

Reply:
License text updated